bff-tiltaksarrangor/src/main/kotlin/no/nav/amt/tiltak/bff/tiltaksarrangor/VeilederController.kt

```diff
@@ -170,7 +170,9 @@ fun ArrangorVeiledersDeltaker.toVeiledersDeltakerDto(endringsmeldinger: List<End
 		deltakerliste = DeltakerlisteDto(
 			id = gjennomforingId,
 			navn = gjennomforingNavn,
-			type = gjennomforingType
+			type = gjennomforingType,
+			startdato = null,
+			sluttdato = null
```

Comment:
Siden det er det samme objektet som brukes her som for endepunktet som brukes for å hente koordinators deltakerlister i oversikten så har jeg rett og slett bare satt disse til null for å slippe å lage ennå et nytt objekt her. Jeg vurderte å sette det til de ordentlige datoene hvis det er kurs her også, men da må jeg hente dem ut fra databasen og ha dem med i ArrangorVeiledersdeltaker også, og det synes jeg blir litt dumt siden datoene egentlig ikke trengs der (ikke ennå hvertfall).

Reply:
Rydder opp i dette også når jeg først er inne og endrer. Det får heller bare bli et ekstra objekt.